src/mappings/ContributionRewardExt/mapping.ts

```diff
@@ -0,0 +1,196 @@
+import { Address, BigInt, Bytes, crypto, store } from '@graphprotocol/graph-ts';
+
+// Import event types from the ContributionRewardExt contract ABI
+import {
+  ContributionRewardExt,
+  NewContributionProposal,
+  ProposalExecuted,
+  RedeemEther,
+  RedeemExternalToken,
+  RedeemNativeToken,
+  RedeemReputation,
+} from '../../types/ContributionRewardExt/ContributionRewardExt';
+
+import * as domain from '../../domain';
+
+import { removeRedeemableRewardOwner } from '../../domain/proposal';
+
+import { shouldRemoveAccountFromUnclaimed, shouldRemoveContributorFromUnclaimed } from '../../domain/reward';
```

Comment:
redundant 
as you are using `import * as domain from '../../domain';`

Reply:
No, this just imports domain/index.ts. Not sufficient.